data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -0,0 +1,99 @@
+# Generated by /usr/libexec/vyos/conf_mode/load-balancing-haproxy.py
+
+global
+    log /dev/log local0
+    log /dev/log local1 notice
+    chroot /var/lib/haproxy
+    stats socket /run/haproxy/admin.sock mode 660 level admin
+    stats timeout 30s
+    user haproxy
+    group haproxy
+    daemon
+
+{% if global_parameters is vyos_defined %}
+{%     if global_parameters.max_connections is vyos_defined %}
+    maxconn {{ global_parameters.max_connections }}
+{%     endif %}
+
+    # Default SSL material locations
+    ca-base /etc/ssl/certs
+    crt-base /etc/ssl/private
+
+{%     if global_parameters.tls.ssl_bind_ciphers is vyos_defined %}
+    ssl-default-bind-ciphers {{ global_parameters.tls.ssl_bind_ciphers | join(':') | upper }}
+{%     else %}
+    # https://ssl-config.mozilla.org/#server=haproxy&version=2.6.12-1&config=intermediate&openssl=3.0.8-1&guideline=5.6
+    ssl-default-bind-ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384
+{%     endif %}
+    ssl-default-bind-ciphersuites TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256
+{%     if global_parameters.tls.tls_version_min is vyos_defined('1.3') %}
+    ssl-default-bind-options force-tlsv13
+{%     else %}
+    ssl-default-bind-options ssl-min-ver TLSv1.2 no-tls-tickets
+{%     endif %}
+{% endif %}
+
+defaults
+    log     global
+    mode    http
+    option  httplog
+    option  dontlognull
+    timeout connect 10s
+    timeout client  50s
+    timeout server  50s
+    errorfile 400 /etc/haproxy/errors/400.http
+    errorfile 403 /etc/haproxy/errors/403.http
+    errorfile 408 /etc/haproxy/errors/408.http
+    errorfile 500 /etc/haproxy/errors/500.http
+    errorfile 502 /etc/haproxy/errors/502.http
+    errorfile 503 /etc/haproxy/errors/503.http
+    errorfile 504 /etc/haproxy/errors/504.http
+
+# Frontend
+{% if server is vyos_defined %}
+{%     for front, front_config in server.items() %}
+frontend {{ front }}
+    bind {{ front_config.listen_address if front_config.listen_address if vyos_defined else '' }}:{{ front_config.port }}
```

Comment:
Do we really need the `else` here?

Reply:
Fixed!

Reply:
`*` expresses to `any` IP address - we might wan't to make it a CLI completion helper option. So user selects either explicitly `any` or a distinct IP address

Reply:
I mean for template, should we remove **else** or use **else '*'**
```
bind {{ front_config.listen_address if front_config.listen_address if vyos_defined }}:{{ front_config.port }} {{ ssl_front }}
```
or
```
bind {{ front_config.listen_address if front_config.listen_address if vyos_defined else '*' }}:{{ front_config.port }} {{ ssl_front }}
```

Reply:
Update. 
I set `*` by default is listen address is not defined